lib/ssh_scan/api.rb

```diff
@@ -0,0 +1,48 @@
+require 'sinatra/base'
+require 'sinatra/namespace'
+require 'ssh_scan/version'
+require 'ssh_scan/policy'
+require 'ssh_scan/scan_engine'
+require 'json'
+
+module SSHScan
+  class API < Sinatra::Base
```

Comment:
Sinatra::Base's run! has the ability to specify port (https://github.com/sinatra/sinatra/blob/master/lib/sinatra/base.rb). I used that above.